net-p2p/syncthing/metadata.xml

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="person">
-    <email>sautier.louis@gmail.com</email>
-    <name>Louis Sautier</name>
-  </maintainer>
   <maintainer type="project">
```

Comment:
Whoever merges this, please remember to also remove proxy-maint along with the last proxied maintainer.